gh-code-scanning

```diff
@@ -150,9 +150,9 @@ class GithubRepository:
                 # Check for primary rate limits. GitHub indicates a primary rate limit through the use of
                 # a 403 status and a set of particular headers.
                 # https://docs.github.com/en/rest/overview/resources-in-the-rest-api#checking-your-rate-limit-status
-                if http_status == '403' and http_headers['X-Ratelimit-Remaining'] == 0:
-                    log.warning('GitHub primary rate limit reached; sleeping for %d seconds.', seconds)
-                    time.sleep(seconds)
+                if http_status == '403' and http_headers['X-Ratelimit-Remaining'] == '0':
+                    log.warning('GitHub primary rate limit reached; sleeping for %s seconds.', seconds)
+                    time.sleep(int(seconds))
```

Comment:
`seconds` (defined in line 136) should always be an integer, but wrapping it with `int()` isn't hurting anything.